app/ui-message/client/message.js

```diff
@@ -73,6 +73,13 @@ const renderBody = (msg, settings) => {
 	const isSystemMessage = MessageTypes.isSystemMessage(msg);
 	const messageType = MessageTypes.getType(msg) || {};
 
+	const result = msg.msg.match(/(^\$kojistart\$)(.+)(\$kojiend\$)$/);
```

Comment:
Instead of putting this logic here, can we use the existing functionality of `MessageTypes` to check and render it like this?